docs/tutorial/notebook/Inference_on_a_custom_data.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>...display what <s>the</s> <s>content</s> <code>data...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done.</p>